# Obstacle waypoint in map coordinates
int32 id
float64 x_center
float64 y_center
float64 x_left
float64 y_left
float64 x_right
float64 y_right
float64 x_start
float64 y_start
float64 x_end
float64 y_end
